src/backend/cpu/Event.hpp

```diff
@@ -11,6 +11,7 @@
 #include <queue.hpp>
 
 #include <common/EventBase.hpp>
+#include <af/event.h>
 
 #include <atomic>
 #include <thread>
```

Comment:
I don't see any object from atomic or thread being used in the header. If these system headers are not required in `Event.hpp` we should move them into `Event.cpp` if they required in that file.